core/src/main/scala/org/apache/spark/metrics/sink/GraphiteSink.scala

```diff
@@ -20,9 +20,8 @@ package org.apache.spark.metrics.sink
 import java.util.{Locale, Properties}
 import java.util.concurrent.TimeUnit
 
-import com.codahale.metrics.MetricRegistry
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
 import com.codahale.metrics.graphite.{Graphite, GraphiteReporter, GraphiteUDP}
-
```

Comment:
This empty line is one of the required Apache Spark coding styles.
```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/core/src/main/scala/org/apache/spark/metrics/sink/GraphiteSink.scala:25:0: There should at least one a single empty line separating groups 3rdParty and spark.
```

You can run the code style checker by the following.
```
$ dev/scalastyle
```